Tasks/AzureRmWebAppDeployment/CompressionUtility.ps1

```diff
@@ -1,5 +1,4 @@
-﻿. $PSScriptRoot/Utility.ps1
-
+﻿
 function UnzipWebDeployPkg
```

Comment:
As discussed with Subbu, the temp path that you are using to extract, may not be a good idea. There are chances you will hit the max file path size limit when you extract zip with deep hierarchy.